sdk/schemaregistry/azure-schemaregistry/tests/async_tests/recordings/test_schema_registry_async.test_get_schema_errors.yaml

```diff
@@ -0,0 +1,27 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json; serialization=Avro
+      User-Agent:
+      - azsdk-python-azureschemaregistry/1.0.0 Python/3.9.0 (Windows-10-10.0.22000-SP0)
+    method: GET
+    uri: https://fake_resource.servicebus.windows.net/$schemaGroups/$schemas/fakeschemaid?api-version=2021-10
+  response:
+    body:
+      string: '{"Code":400,"Detail":"SubCode=40000, UnknownType:The request is invalid.
+        [MGResponseHttpError=BadRequest]. TrackingId:4d975522-ca3a-427d-87fd-e130a109baff_G2,
+        SystemTracker:fake_resource.servicebus.windows.net/:$schemaGroups\/$schemas\/fakeschemaid,
+        Timestamp:2022-02-18T22:19:56"}'
```

Comment:
I did not expect this body, hmm.

Reply:
I was expecting something like [this](https://github.com/Azure/azure-sdk-for-js/blob/c50543f23a06f006ac41d79ca4af023f56276326/sdk/schemaregistry/schema-registry/recordings/node/schemaregistryclient/recording_fails_to_get_schema_when_no_schema_exists_with_given_id.json#L24-L29)

Reply:
Just tested with West US 2 and got the same status codes as East US, so I don't think that's the problem. 🤔 Maybe we can see what the other languages get?

Reply:
@deyaaeldeen - same status codes as above with West US.

Reply:
just added error code checks, and these look the same as JS.